### PR TITLE
[W5.7][T13-3]Qi Gongjie

### PR DESCRIPTION
**Pull request by Gongjie663746**
Add CountCommand.

### CODE REVIEW

**Comment by HaydenPhillips** [comment]
Nice work!

File: src/seedu/addressbook/commands/CountCommand.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+/**
+ * Count the number of all persons in the address book to the user.
+ */
+public class CountCommand extends Command {
```
Comment (NancyQuris) [inline src/seedu/addressbook/commands/CountCommand.java:11]: Seen! Great work!

File: src/seedu/addressbook/commands/CountCommand.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+
```
Comment (parvathy-sudhir-pillai) [inline src/seedu/addressbook/commands/CountCommand.java:3]: Stray line break? It is a good practice to check the diff of the PR and fix any problems before submitting it.

**Review by parvathy-sudhir-pillai — APPROVED** [review]
Good effort on this PR. In particular, good job on keeping the PR relatively clean

Suggestions to improve further:
* update user and dev docs
* update jUnit and I/O tests to test the new functionality
* Pay closer attention to the coding standard

Please close the PR after viewing comments.

**Pull request closed without merging**